web/src/test/java/com/replica/StreetEdgeExporterTest.java

```diff
@@ -26,29 +30,40 @@
 
     @Test
     public void testExportEndToEnd() throws IOException {
-        cli.run("export", TEST_GRAPHHOPPER_CONFIG_PATH);
         CSVFormat format = StreetEdgeExporter.CSV_FORMAT;
-        File expectedOutputLocation = new File(GRAPH_FILES_DIR + "street_edges.csv");
+        File expectedOutputLocation = new File(EXPORT_FILES_DIR + "street_edges.csv");
         CSVParser parser = CSVParser.parse(expectedOutputLocation, StandardCharsets.UTF_8, format);
         List<CSVRecord> records = parser.getRecords();
         assertEquals(1102357, records.size());
+        Helper.removeDir(new File(EXPORT_FILES_DIR));
     }
 
     @Test
     public void testExportSingleRecord() throws Exception {
-        // TODO copied from ExportCommand
+        // Load graph and run OSM parsing step to pull tag info from OSM
         GraphHopper configuredGraphHopper = graphHopperManaged.getGraphHopper();
+        CustomGraphHopperGtfs gh = (CustomGraphHopperGtfs) configuredGraphHopper;
+        gh.collectOsmInfo();
 
-        // Load OSM info needed for export from MapDB database file
-        DB db = DBMaker.newFileDB(new File("transit_data/osm_info.db")).readOnly().make();
-        Map<Long, Map<String, String>> osmIdToLaneTags = db.getHashMap("osmIdToLaneTags");
-        Map<Integer, Long> ghIdToOsmId = db.getHashMap("ghIdToOsmId");
-        Map<Long, List<String>> osmIdToAccessFlags = db.getHashMap("osmIdToAccessFlags");
-        Map<Long, String> osmIdToStreetName = db.getHashMap("osmIdToStreetName");
-        Map<Long, String> osmIdToHighway = db.getHashMap("osmIdToHighway");
+        // Hacky override used to populate GH ID -> OSM ID map, normally stored in-memory during nationwide export
+        // All other OSM-derived info is parsed in above call to collectOsmInfo()
+        Map<Integer, Long> ghIdToOsmId = Maps.newHashMap();
```

Comment:
Currently, all of this gets done during the graphhopper import step that runs in our nationwide export command. The reason I need to hackily simulate it here is that we're using graphhopper files that were already imported during the unit test setup. The mapping of gh ID <-> OSM ID was generated during that import, but is no longer stored in (mapdb) files, so we have no way to access it here without regenerating it. 

In the nationwide export command, we're still OK though, because we keep the imported graphhopper files in memory, and therefore don't lose this ID mapping info